projects/com.oracle.truffle.llvm.types/src/com/oracle/truffle/llvm/types/LLVMFunctionDescriptor.java

```diff
@@ -45,6 +46,7 @@
 import com.oracle.truffle.api.nodes.RootNode;
 import com.oracle.truffle.llvm.types.memory.LLVMStack;
 
+@ValueType
 public final class LLVMFunctionDescriptor implements TruffleObject, Comparable<LLVMFunctionDescriptor> {
```

Comment:
since it's a TruffleObject, I assumed that it gets handed out to other languages.